plugin/tooling.py

```diff
@@ -161,6 +167,8 @@ def run(self, base_package_name: str) -> None:
         self.view.assign_syntax("Packages/JSON/JSON.sublime-syntax")
         self.writeline("{")
         for k, v in sorted(properties.items()):
+            if not isinstance(v, dict):
```

Comment:
I don’t think this check is actually necessary now. But better safe than sorry I suppose :)